File: pinyougou-seckill-interface/src/main/java/com/pinyougou/seckill/service/SeckillGoodsService.java
package com.pinyougou.seckill.service;

import com.pinyougou.model.SeckillGoods;

import java.util.List;

/***
 *
 * @Author:shenkunlin
 * @Description:itheima
 * @date: 2018/9/28 16:06
 *
 ****/
public interface SeckillGoodsService {


    /***
     * 根据ID查询秒杀商品
     * @param id
     * @return
     */
    SeckillGoods getOne(Long id);

    /****
     * 秒杀商品列表查询
     * @return
     */
    List<SeckillGoods> list();

}

File: pinyougou-seckill-web/src/main/webapp/js/service/seckillGoodsService.js
app.service('seckillGoodsService',function ($http) {

    //下单
    this.add=function (id) {
        return $http.get('/seckill/order/add.shtml?id='+id);
    }

    //根据DI查询商品详情
    this.getOne=function (id) {
        return $http.get('/seckill/goods/one.shtml?id='+id);
    }

    //秒杀列表
    this.list=function () {
        return $http.get('/seckill/goods/list.shtml');
    }

})